src/test/java/com/epam/rp/api/tests/ApiUserTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.rp.api.tests;
+
+import com.epam.rp.utils.Logger;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertNotNull;
+
+public class ApiUserTest extends ApiBaseTest {
+//    protected static String apiToken;
+//    protected static String baseUrl;
```

Comment:
It's better not leave commented code, feel free to remove it in case it is not required.